### ISSUE TITLE
[Gameplay] Implement basic city traffic

### ISSUE THREAD

**Issue by codenamecpp** [codenamecpp/carnage3d#25]
This includes:
- [ ] Traffic cars generation and navigation
- [x] Pedestrians generation and navigation

Traffic cars and peds should be destroyed over time when they go out of screen.

**Comment by codenamecpp** [comment]
Now there is generation and basic navigation for traffic peds. They randomly walking and panic if they are on fire :)

**Comment by neuromancer** [comment]
It works!, however, in the demo level, pedestrian are only generated on the "lower parts" of the city. When you start, you can't see any one, but if you drive downhill, there are plenty of them.

**Comment by codenamecpp** [comment]
Yeah, there are some troubles with SANB.CMP, I'll try to figure it out

**Comment by codenamecpp** [comment]
Ok, seems to fixed

**Comment by neuromancer** [comment]
Yes, confirmed!
